library/Vanilla/Permissions.php

```diff
@@ -274,6 +293,21 @@ public function setBanned($isBanned) {
     }
 
     /**
+     * Get the flag to determine if all permission checks are being overridden.
+     *
+     * @param bool|null $overrideAll
+     * @return $this
+     */
+    public function setOverrideAll($overrideAll) {
+        if ($overrideAll !== null) {
+            $overrideAll = boolval($overrideAll);
```

Comment:
This could be used by any developer making addons. It'd be great if we could count on everyone to always do the right thing, but I think it's good to correct for simple "mistakes".